projects/ad9213_dual_ebz/Readme.md

```diff
@@ -0,0 +1,8 @@
+# AD9213-DUAL-EBZ HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/eval-ad9213)
+  * Parts : [12-Bit, 10.25 GSPS, JESD204B, RF Analog-to-Digital Converter](https://www.analog.com/ad9213)
+  * Project Doc: https://wiki.analog.com/ad9213
```

Comment:
for this project, the project doc is at: https://wiki.analog.com/resources/eval/user-guides/ad9213_dual_ebz/ and the HDL is at: https://wiki.analog.com/resources/eval/user-guides/ad9213_dual_ebz/ad9213_dual_ebz_hdl. Linux driver is under development